### PR TITLE
Theme styling fixes

### PR DESCRIPTION
**Pull request by flesher1**
Adds search window functionality for closing the window
Fix some styling in bookmarks and other places for different themes
Fix 'Removes all'button from the advanced search
Fixes issues with word search

### CODE REVIEW

File: src/components/navBar/Search.js [hunk 1]
```diff
@@ -89,6 +89,8 @@ class Search extends React.Component {
                     interactionKind={PopoverInteractionKind.CLICK}
                     position={Position.BOTTOM}
                     isOpen={this.props.open}
```
Comment (blynch11p) [inline src/components/navBar/Search.js]: this could just be `open` now ;)

File: src/components/navBar/SearchFilter.js [hunk 2]
```diff
@@ -130,7 +136,7 @@ class SearchFilter extends React.Component {
                             onChange={this.change508Filter}
                             label={<h5>Only 508 compliant listings</h5>}/>
                     </div>
-                    <button className="btn btn-primary btn-sm float-right" onClick={this.clearAllFilter}><Icon icon="cross" />Clear All Filters</button>
+                    <button className="btn btn-primary btn-sm float-right" onClick={this.clearAllFilter}><Icon icon="cross" />Clear Advanced Seach</button>
```
Comment (blynch11p) [inline src/components/navBar/SearchFilter.js]: Seach => Search

**Comment by mleeBoeing** [comment]
When clicking on Advanced Search, the filter window no longer acts like a Popover and does not follow when you scroll down/up.
image

Should look like this:
image

**Comment by mannyrivera2010** [comment]
:+1:

**Pull request merged**